cluster/clean.sh

```diff
@@ -70,6 +70,7 @@ for i in ${namespaces[@]}; do
         _kubectl -n ${i} delete configmaps -l ${label}
         _kubectl -n ${i} delete secrets -l ${label}
         _kubectl -n ${i} delete jobs -l ${label}
+        _kubectl -n ${i} delete csr --all
```

Comment:
csr's don't have the kubevirt.io label?

Reply:
No, actually not easy to add them because we use the k8s client library which hides all these details. However, I don't think that we should delete any CSRs at all. k8s itself cleans them up and we don't reuse them in any way. That would work nice for all types of providers we support.

Reply:
Done. No CSRs are deleted anymore, since they are cleaned up by k8s itself anyway.